src/services/item/plugins/invitation/test/index.test.ts

```diff
@@ -539,6 +552,118 @@ describe('Invitation Plugin', () => {
+      const response = await app.inject({
+        method: HttpMethod.Post,
+        url: `${ITEMS_ROUTE_PREFIX}/${v4()}/invitations/upload-csv`,
+        payload: form,
+        headers: form.getHeaders(),
+      });
+
+      expect(response.statusCode).toBe(StatusCodes.UNAUTHORIZED);
+    });
+
+    describe('Signed In', () => {
+      let item, mailerService;
+
+      beforeEach(async () => {
+        actor = await saveMember();
+        mockAuthenticate(actor);
+        ({ item } = await testUtils.saveItemAndMembership({ member: actor }));
+        mailerService = resolveDependency(MailerService);
+      });
+
+      it('upload csv successfully', async () => {
+        const mockSendEmail = jest.spyOn(mailerService, 'sendEmail');
+        const form = createFormData('users.csv');
+        const response = await app.inject({
+          method: HttpMethod.Post,
+          url: `${ITEMS_ROUTE_PREFIX}/${item.id}/invitations/upload-csv`,
+          payload: form,
+          headers: form.getHeaders(),
+        });
+        expect(response.statusCode).toEqual(StatusCodes.OK);
+
+        for (const inv of response.json().invitations) {
+          expect(mockSendEmail).toHaveBeenCalledWith(
+            expect.anything(),
+            inv.email,
+            expect.anything(),
+            expect.anything(),
+            expect.anything(),
+          );
+        }
+      });
+    });
+  });
+
+  describe('POST /:itemId/invitations/upload-csv-template', () => {
+    it('throws if signed out', async () => {
+      const form = createFormData('users-groups.csv');
+      const response = await app.inject({
+        method: HttpMethod.Post,
+        url: `${ITEMS_ROUTE_PREFIX}/${v4()}/invitations/upload-csv-template`,
+        query: { templateId: v4() },
+        payload: form,
+        headers: form.getHeaders(),
+      });
+
+      expect(response.statusCode).toBe(StatusCodes.UNAUTHORIZED);
+    });
+
+    describe('Signed In', () => {
+      let item, mailerService;
+
+      beforeEach(async () => {
+        actor = await saveMember();
+        mockAuthenticate(actor);
+        ({ item } = await testUtils.saveItemAndMembership({ member: actor }));
+        mailerService = resolveDependency(MailerService);
+      });
+
+      it('upload csv successfully', async () => {
+        const { item: templateItem } = await testUtils.saveItemAndMembership({ member: actor });
+        const mockSendEmail = jest.spyOn(mailerService, 'sendEmail');
+        const form = createFormData('users-groups.csv');
+        const response = await app.inject({
+          method: HttpMethod.Post,
+          url: `${ITEMS_ROUTE_PREFIX}/${item.id}/invitations/upload-csv-template`,
+          query: { templateId: templateItem.id },
+          payload: form,
+          headers: form.getHeaders(),
+        });
+        expect(response.statusCode).toEqual(StatusCodes.OK);
+        for (const group of response.json()) {
+          for (const inv of group.invitations) {
```

Comment:
You only test invitations, should we also test the memberships created ? Are there any in this test ?